pkg/localstore/mode_set.go

```diff
@@ -365,6 +365,10 @@ func (db *DB) setUnpin(batch *leveldb.Batch, addr swarm.Address) (gcSizeChange i
 	item.StoreTimestamp = i.StoreTimestamp
 	item.BinID = i.BinID
 	item.BatchID = i.BatchID
+	err = db.pullIndex.DeleteInBatch(batch, item)
```

Comment:
Same

Reply:
so here if an item is in the pushIndex, it should not affect the reserve, this is why it's moved above the pushIndex check.